2018-westphal-antpc.tex

```diff
@@ -177,27 +172,28 @@ \subsection{Electrowinning}
+Users provide the separations archetype with a feed stream and facility efficiencies.  
+Each waste stream requires a material balance over voloxidation, electroreduction, electrorefining and electrowinning. Main 
 waste streams are metallic waste, ceramic waste from electrowinning and electroreduction, and vitrified waste. Vitrified 
-waste contains the majority of TRUs, Sr, and rare-earth elements. The elemental separation efficiencies are 
+waste contains the majority of TRUs, Sr, and rare-earth elements. Elemental separation efficiencies are 
 determined through theoretical material balance determined by the NEA and Hermann et al \cite{flowsheet_1998,herrmann_separation_2010}. 
 The simple simulation using a separations facility was run to verify the table of efficiencies input to \Cyclus. A scenario was 
 created of one separations facility with a feed of five year cooled spent LWR 
 fuel at a burn-up of 45 \gls{GWd} per \gls{MTIHM} to 
 match results seen in \cite{flowsheet_1998}.
 
-A pyroprocessing facility can be modeled with the separations archetype at low fidelity 
-by a dedicated archetype. The goal for Pyre is to include facility configuration parameters and 
-their respective effects on the efficiency table. Any material not separated by the four processes is categorized as
-leftover commodity, ensuring material conservation is maintained. A test will also be used to verify isotopic 
-efficiencies add up to 1. Efficiencies also vary according to the feed stream resulting in different 
-waste streams for LWR and FR fuels, for example. Multiple material choices exist for anodes and cathodes as well as other 
-design choices that require consideration. 
+A pyroprocessing facility can be modeled with the separations archetype at low fidelity. 
+The goal for PyRe is to include facility configuration parameters and their respective effects on the efficiency table. 
+Any material not separated by the four processes is categorized as leftover commodity, ensuring material conservation is maintained. 
+A test will also be used to verify isotopic efficiencies add up to 1. 
+Efficiencies also vary according to the feed stream resulting in different waste streams for LWR and FR fuels, for example. 
+Multiple material choices exist for anodes and cathodes as well as other design choices that require consideration. 
 
 \subsection{Parameters}
-Facility configuration parameters customize the pyroprocessing archetype. Facility designs vary in 
-multiple aspects which affect the throughput and efficiencies of different waste streams. Variables have been described above through material balance of each sub-process and their system parameters. Table \ref{tab:params} consists of the  derived from the previously mentioned process parameters and material flow. 
+Facility configuration parameters customize the pyroprocessing archetype. 
```

Comment:
This paragraph reads more like a bulleted list than a paragraph. Try reading it aloud and then rewrite the main idea in conversational english.